cmd/helm/create.go

```diff
@@ -54,6 +55,10 @@ something like this:
 do not exist, Helm will attempt to create them as it goes. If the given
 destination exists and there are files in that directory, conflicting files
 will be overwritten, but other files will be left alone.
+
+The created or scaffold chart contains deployment, ingress and service
```

Comment:
It might be best to reference the Helm docs here instead as that is more helpful to the end-user. Here's a suggested amendment to this paragraph:

"The chart scaffolder by invoking this command contains a Deployment, Ingress and a Service. To use other Kubernetes resources with your chart, refer to [The Chart Template Developer's Guide](https://helm.sh/docs/chart_template_guide)."

Reply:
get.

Reply:
@bacongobbler done